libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -504,3 +504,7 @@ singleton AP_Follow method get_target_location_and_velocity boolean Location'Nul
 singleton AP_Follow method get_target_location_and_velocity_ofs boolean Location'Null Vector3f'Null
 singleton AP_Follow method get_target_heading_deg boolean float'Null
 
+include AC_AttitudeControl/AC_AttitudeControl.h depends APM_BUILD_TYPE(APM_BUILD_ArduPlane)||APM_BUILD_COPTER_OR_HELI
+singleton AC_AttitudeControl depends APM_BUILD_TYPE(APM_BUILD_ArduPlane)||APM_BUILD_COPTER_OR_HELI
+singleton AC_AttitudeControl method get_rpy_PDmod boolean Vector3f'Null
```

Comment:
No need to return always true if you do this:
```suggestion
singleton AC_AttitudeControl method get_rpy_PDmod void Vector3f'Ref
```

Reply:
thanks, fixed